### PR TITLE
Mh/none bump jenkins version to 2.319

### PR DESCRIPTION
**Pull request by mhenschke-atl**

### CODE REVIEW

File: acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/applink/oauth/ThreeLeggedOAuthAcceptanceTest.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ public void setUp() throws Exception {
     @Test
     @Ignore("https://github.com/jenkinsci/acceptance-test-harness/issues/721")
     // This issue is fixed in the v1.109 of acceptance-test-harness but that version is not compatible with current
-    // Jenkins version (v2.289.1) we are using or the LTS (v2.319.1). Jenkins version should be >= v2.323 for v1.109 of
+    // Jenkins version (v2.289.3) we are using or the LTS (v2.319.1). Jenkins version should be >= v2.323 for v1.109 of
```
Comment (mhenschke-atl) [inline acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/applink/oauth/ThreeLeggedOAuthAcceptanceTest.java:78]:
```suggestion
    // Jenkins version (v2.319) we are using or the LTS (v2.319.1). Jenkins version should be >= v2.323 for v1.109 of
```

File: acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/SmokeTest.java [hunk 2]
```diff
@@ -193,7 +193,7 @@ public void testRunFolderCredentialsMultibranch() throws Exception {
     @Test
     @Ignore("https://github.com/jenkinsci/acceptance-test-harness/issues/721")
     // This issue is fixed in the v1.109 of acceptance-test-harness but that version is not compatible with current
-    // Jenkins version (v2.289.1) we are using or the LTS (v2.319.1). Jenkins version should be >= v2.323 for v1.109 of
+    // Jenkins version (v2.319) we are using or the LTS (v2.319.1). Jenkins version should be >= v2.323 for v1.109 of
```
Comment (mhenschke-atl) [inline acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/SmokeTest.java:196]: I think we should merge this first, but now our minimum version is high enough we can address this smoke test failure :smile:

File: acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/applink/oauth/ThreeLeggedOAuthAcceptanceTest.java (hunk 1 above)
Comment (EliSalter-atl) [reply acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/applink/oauth/ThreeLeggedOAuthAcceptanceTest.java:78]: v2.319?

File: readme.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ The plugin streamlines the entire configuration process and removes the need for
 
 ## Requirements
 
-- Jenkins 2.289.1+
+- Jenkins 2.391+
```
Comment (EliSalter-atl) [inline readme.md:29]: 2.319+
```suggestion
- Jenkins 2.319+
```

**Review by EliSalter-atl — APPROVED** [review]

File: acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/applink/oauth/ThreeLeggedOAuthAcceptanceTest.java (hunk 1 above)
Comment (mhenschke-atl) [inline acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/applink/oauth/ThreeLeggedOAuthAcceptanceTest.java:78]:
```suggestion
    // Jenkins version (v2.319) we are using or the LTS (v2.319.1). Jenkins version should be >= v2.323 for v1.109 of
```

**Pull request merged**